onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -811,12 +811,12 @@ def _where_input_wrangler(
         matcher=lambda sample: sample.args[1] == 2,
         reason="fixme: 'bicubic' mode in ORT implemented differently with Torch",
     ),
-    xfail(
+    skip(
```

Comment:
Some float16 test cases passed?

Reply:
No relation with float16 here. I just think it should be skip instead of xfail. But I don't have strong reason.